changelogs/unreleased/pagination.md

```diff
@@ -0,0 +1,3 @@
+## feature/core
+
+* Memtx and vinyl tree indexes supports pagination now.
```

Comment:
- supports -> support
- Missing reference to the ticket.
- Let's be a bit more verbose - not everyone may now what "pagination" is:
   > Introduced pagination support for memtx and vinyl tree indexes. Now, it's possible to resume `pairs` and `select` from the position where the last call stopped (gh-XXXX).

Reply:
Done